packages/core/__tests__/annotations.test.tsx

```diff
@@ -101,7 +101,7 @@ describe('Annotations', () => {
       target: { innerText: 'New Value', innerHTML: 'New Value' }
     })
     await userEvent.click(exam.getByText('Vastaa'))
-    await annotateText(exam, 'Esitä jokin toinen käsitys filosofiasta ja vertaa sitä')
+    await annotateText(exam, ' Esitä jokin toinen käsitys filosofiasta ja vertaa sitä ')
```

Comment:
Tekstikappaleissa on aina alussa ja lopussa tyhjät välilyönnit 🤔 
Näin on ollut aina näemmä.